jenkins-dsl/cassandra_job_dsl_seed.groovy

```diff
@@ -105,6 +105,19 @@ job('Cassandra-template-artifacts') {
             javadocDir 'build/javadoc'
             keepAll false
         }
+        extendedEmail {
+            recipientList('builds@cassandra.apache.org')
+            triggers {
+                stillUnstable {
```

Comment:
stillUnstable is defined as:
// Triggers an email if the build status is "Unstable" for two or more builds in a row.

I think we're going for failure{} and fixed{}? These triggers would send on every build failure - there is an alternate firstFailure{} trigger, so only one email would be sent in the case of repeated build failures. I can't remember if we used regression{} in other jenkinses, but that might suit the needs for this without spamming every failure{} too. I'll see if I can find some old DSL foo to see what we used.

Reply:
Fixed.

> These triggers would send on every build failure - there is an alternate firstFailure{} trigger, so only one email would be sent in the case of repeated build failures.

Let's start with `failure {}`, as no one should be committing on top of a broken HEAD. (In a perfect world, that should be as bad as breaking it to begin with.)
(If/When we change to `firstFailure{}` then `culprits()` looks like it can be removed.)